git_machete/cli.py

```diff
@@ -556,6 +560,16 @@ def launch(orig_args: List[str]) -> None:
                     machete_client.annotate(branch, parsed_cli.annotation_text)
                 else:
                     machete_client.print_annotation(branch)
+        elif cmd == "clean":
+            machete_client.read_definition_file()
+            machete_client.delete_untracked(opt_yes=cli_opts.opt_yes,
+                                            opt_merge=cli_opts.opt_merge,
+                                            opt_no_interactive_rebase=cli_opts.opt_no_interactive_rebase,
+                                            opt_no_edit_merge=cli_opts.opt_no_edit_merge)
+            machete_client.delete_unmanaged(opt_yes=cli_opts.opt_yes)
+            if '--checkout-my-github-prs' in parsed_cli:
```

Comment:
Hmm should the options be checked against `parsed_cli` with `--` or without? below there's e.g.

```
            if 'draft' in parsed_cli and github_subcommand != 'create-pr':
```

Reply:
I.e. without the leading `--` for `draft`

Reply:
Oh, great catch, indeed it should be without leading **--** and also **-** should be replaced with **_**

Reply:
Pls check if there are any other typos like that